public/Sjofn/practice/sjofn4.html

```diff
@@ -0,0 +1,32 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Practice 4</title>
+</head>
+<body>
+<!--<button onclick="instruction1()">Open console and click me</button>-->
+</body>
+<script>
+    function getDatetime() {
+        console.log(new Date());
```

Comment:
`new Date()` contains unformatted object:

> Thu Feb 20 2020 19:41:12 GMT+0200 (Восточная Европа, стандартное время)

But there is a way to format the datetime and transform it to a readable string before printing it - `new Date().toLocaleString()`:

> 20.02.2020, 19:41:01

P.S.: it's just for your information, - no need to change anything here if you don't want to.